mariadb.html.md.erb

```diff
@@ -112,12 +112,12 @@ See [Migrate from legacy MariaDB to MariaDB Ent](./migrate-mariadb-to-mariadbent
 
 
 ## <a id='backup'></a> Backup
-Within our MariaDB service you can start a backup and also restore these afterwards. 
-For more information, check the [service backup](../devguide/services/backups.html) guide.
+Within your MariaDB service instance you can start a backup and also restore these afterwards.
+For more information about this Developer Portal feature, check the [service backup](../devguide/services/backups.html) guide.
```

Comment:
It's not a developer portal feature, it can also by done via CLI with the swisscom cli plugin :)

Reply:
fixed. Thanks.